openedx/features/caliper_tracking/decorators.py

```diff
@@ -0,0 +1,23 @@
+from functools import wraps
+
+from .processor import CaliperProcessor
+
+
+def transform_caliper(logging_func):
+    """
+    Decorator to transform events into caliper transformed events.
+    """
+
+    @wraps(logging_func)
+    def transform(event):
+        caliper_event = None
+
+        if event and '/' not in event['event_type']:
```

Comment:
What does this check do exactly?

Reply:
There are many server event which we don't need, this check won't send these for caliparization for example: 
`"event_type": "/courses/course-v1:edX+DemoX+Demo_Course/xblock/block-v1:edX+DemoX+Demo_Course+type@problem+block@ex_practice_3/handler/xmodule_handler/problem_check"`